### ISSUE TITLE
Use Case Revision

### ISSUE THREAD

**Issue by TurkerErdem99** [bounswe/bounswe2022group9#236]
Use cases are outdated and need to be revised. Keep in mind that everything related to "bidding" actions should be removed since bidding action is removed from the project. Moreover, "signup with Google" actions are changed and use cases should be edited accordingly.

**Comment by burakaktan** [comment]
I checked Use Case diagrams. I decided to have a TODO list for track the changes easier.

TODO:
-[ ] Change "Google" related parts in [Register(Sign Up) Use Case] (https://github.com/bounswe/bounswe2022group9/wiki/Use-Case-Diagram#register-sign-up) , since the group is decided email-password authentication with verified email is enough.
-[ ] Delete "Bidding Use Case" , since bidding is an optional feature and decided not to be implemented in group meeting on 9th October of 2022
-[ ] Delete "Block User Use Case"
